docs/.vuepress/config.js

```diff
@@ -47,6 +57,42 @@ module.exports = {
       ])
       (token.attrGet('href'));
     });
+
+    markdown.use(markdownItForInline, 'youtube-link', 'text', (tokens, index) => {
+      pipe([
+        getYoutubeDocumentId,
+        map(id => {
+          tokens[index] = {
```

Comment:
Not setting up `token = tokens[index]` because overriding a variable reference doesn't work as intended. :roll_eyes: